arrocke/gloss-translation
# fix: darken hero image more

## What has changed

darkens hero image to improve contrast with the text.



## Connected Issues

closes #360 



## QA Steps

load landing page and check for color contrast. You can upload the image to https://www.brandwood.com/a11y/ to check for sufficient contrast
